src/Shared/EventArgsFormatting.cs

```diff
@@ -219,7 +219,9 @@ internal static string FormatEventMessage
             string logOutputProperties
         )
         {
-            StringBuilder format = new StringBuilder();
+            // capacity is the longest possible path through the below
+            // to avoid reallocating while constructing the string
+            StringBuilder format = StringBuilderCache.Acquire(48);
```

Comment:
I think this could theoretically be up to 51 characters long? Not that it should really matter...